### PR TITLE
Use npm ci instead of npm install

### PR DESCRIPTION
**Pull request by leafac**
https://docs.npmjs.com/cli/ci.html

### CODE REVIEW

**Comment by andymckay** [comment]
Thank you @leafac, I think we've got this in #56.

**Comment by zachguo** [comment]
@andymckay it's not the same issue actually

**Comment by andymckay** [comment]
Oh thanks for spotting my mistake @zachguo, this is about a different file.

**Comment by andymckay** [comment]
@leafac how do you feel about changing that line?

File: ci/node.js.yml [hunk 1]
```diff
@@ -17,8 +17,8 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: npm install, build, and test
```
[the remark this answers is not in the document]
Comment (leafac) [reply ci/node.js.yml]: No problem.

**Comment by leafac** [comment]
@andymckay done.

**Comment by andymckay** [comment]
Thanks @leafac

**Pull request merged**